.env.test

```diff
@@ -1,7 +1,7 @@
 # define your env variables for the test env here
 KERNEL_CLASS='App\Kernel'
 ILIOS_SECRET='$ecretf0rt3st'
-SYMFONY_DEPRECATIONS_HELPER=999999
+SYMFONY_DEPRECATIONS_HELPER=max[total]=9999&max[self]=0&verbose=1
```

Comment:
what in tarnation?!

Reply:
explained here: https://symfony.com/doc/current/components/phpunit_bridge.html#internal-deprecations